api/src/main/java/jakarta/data/page/CursoredPage.java

```diff
@@ -120,7 +120,7 @@
  * &#64;OrderBy("birthYear")
  * &#64;OrderBy("id")
  * CursoredPage&lt;Customer&gt; getTopBuyers(int minOrders, float minSpent,
- *                                     PageRequest pageRequest, Order&lt;Customer&gt;} sort);
+ *                                     PageRequest pageRequest);
```

Comment:
In this case, we overdid it and added a `Order` parameter that doesn't make sense -- it is already ordered by its unique identifier, making any lesser priority sorting unreachable.